tests/test_runner.py

```diff
@@ -86,18 +86,18 @@ def fcn(analysis):
 
         runner.run(fcn)
 
-    def test_exception_raised_when_extra_strand_data_present(self):
-        """ Ensures that protected attributes can't be set
-        """
-        with self.assertRaises(twined.exceptions.StrandNotFound) as error:
-            Runner(
-                twine="{}", configuration_values={},
-            )
-
-        self.assertIn(
-            "Source data is provided for 'configuration_values' but no such strand is defined in the twine",
-            error.exception.args[0],
-        )
+    # def test_exception_raised_when_extra_strand_data_present(self):
+    #     """ Ensures that protected attributes can't be set
+    #     """
+    #     with self.assertRaises(twined.exceptions.StrandNotFound) as error:
+    #         Runner(
+    #             twine="{}", configuration_values={},
+    #         )
+    #
+    #     self.assertIn(
+    #         "Source data is provided for 'configuration_values' but no such strand is defined in the twine",
+    #         error.exception.args[0],
+    #     )
```

Comment:
Is this no longer compatible with the latest version of twined?

Reply:
I think it's not valid anymore with the changes I made (I've allowed e.g. configuration values to be passed into *the Runner* even if the twine doesn't allow it to make the CLI simpler; note: the twine still behaves the same and won't allow extra strand data to be used)

Reply:
Shall I remove it?

Reply:
I'll do it, I've got it right here